configs/components/oifs/oifs.yaml

```diff
@@ -856,6 +875,61 @@ choose_version:
                 add_outdata_files:
                         ICMUA: ICMUA
 
+
+# If 1, use Coddington spectrum which reduced stratospheric warm bias
+# Only available in 43r3-v2
+choose_solarspectrum:
+        True:
+                add_namelist_changes:
+                        fort.4:
+                                NAERAD:
+                                        NSOLARSPECTRUM: 1
+
+# There are two different O3 concentrations in OpenIFS:
+# One is used for output, and one that is used by radiation scheme
+# By default, the two are NOT the same.
+# By default, prognostic O3 from the Cariolle scheme is written to output
+# but climatology is used by radiation scheme
+#
+# Here the scheme can be changed in two ways:
+# cmip6: Use O3 from CMIP6 (input4MIP) for picontrol, historical or scenarios
+# cariolle: Use the prognostic O3 from Cariolle in radiation
+choose_o3_scheme:
+        default:
+                add_namelist_changes:
+                        fort.4:
+                                NAERAD:
+                                        NGHGRAD: 21
```

Comment:
@JanStreffing: This is adding to the `fort.4` `NGHGRAD: 21` for `AWICM3` simulations that did not have this parameter (`v3.0`, `v3.1`, `frontiers-xios`). If that's okay we can leave it as it is. If it's not okay, we will need to set `o3_scheme: null` in `awicm3.yaml`, so that none of this cases are selected.

Reply:
You can see the changes in the `fort.4` in the work folder: https://github.com/esm-tools/esm_tools/actions/runs/3203196145/jobs/5232999271#step:8:666

Reply:
The default value in the source code it's 21 anyway, so we will keep the default configuration as it currently is.